python/src/LeastSquaresExpansion_doc.i.in

```diff
@@ -0,0 +1,62 @@
+%feature("docstring") OT::LeastSquaresExpansion
+"L2 approximation on an orthonormal basis using least-squares and a fixed basis.
+
+Available constructors:
+    LeastSquaresExpansion(*inputSample, outputSample, distribution*)
+
+    LeastSquaresExpansion(*inputSample, weights, outputSample, distribution*)
+
+    LeastSquaresExpansion(*inputSample, outputSample, distribution, basis, basisSize, methodName*)
+
+    LeastSquaresExpansion(*inputSample, weights, outputSample, distribution, basis, basisSize, methodName*)
+
+Parameters
+----------
+inputSample : 2-d sequence of float of size :math:`n>0`
+    Input sample.
+weight : sequence of float of size :math:`n>0`, optional
+    Input weights. If not given, the default value is a uniform weight equals to :math:`1/n`.
```

Comment:
same here
You have 1 in the code, which seems more logical

Reply:
The correct value is `1/n` as the weights have to sum to 1